frontend/js/marc-record-merge-postmerge-service.js

```diff
@@ -447,6 +462,26 @@ export function removeExtra035aFromMerged(preferredRecord, otherRecord, mergedRe
   };
 }
 
+export function add080VersionCode (preferredRecord, otherRecord, mergedRecordParam) {
+  const mergedRecord = new MarcRecord(mergedRecordParam);
+
+  mergedRecord.fields.filter(field => field.tag === '080')
+    .forEach(field => {
+      if (field.subfields.some(({code}) => code === '2')) {
+        return;
+      }
+      let index = field.subfields.findIndex(sub => !isNaN(sub.code) && sub.code >= '2');
```

Comment:
1. Always use `const` unless it's necessary to use `let`
2. Treating strings as numbers is not good practise (Even though possible in Javascript). Change this to `Number(sub.code) > 2` (If `sub.code` is not numeric, the constructor will return `NaN` and therefore the comparison will return `false`. Also, use `>` instead `>=` since we are only interested in values greater than `2` (+ This line is never reached if subfield `2` exists in the field)